src/ngraph/op/divide.cpp

```diff
@@ -129,3 +184,9 @@ void op::v1::Divide::generate_adjoints(autodiff::Adjoints& adjoints, const Outpu
     adjoints.add_delta(x, delta / y);
     adjoints.add_delta(y, -delta * shared_from_this() / y);
 }
+
+bool op::v1::Divide::evaluate(const EvaluatorTensorVector& outputs,
+                              const EvaluatorTensorVector& inputs)
```

Comment:
Why a new class for Tensor? What is wrong with runtime/tensor.hpp and runtime/host_tensor.hpp? Those already exist for this specific purpose and should be used instead of creating new classes.